ethcore/src/ethereum/ethash_eng.rs

```diff
@@ -23,19 +23,23 @@ use ethereum_types::{H256, H64, U256};
 use ethjson;
 use hash::{KECCAK_EMPTY_LIST_RLP};
 use rlp::Rlp;
-use types::header::Header;
-use types::BlockNumber;
+use types::{
```

Comment:
why `ethash_eng.rs`?

Reply:
Honestly no good reason: my IDE was confused by a crate and module with the same name. OTOH it’s odd that we have an engine outside the engines folder so my plan was to move it anyway.